src/main/resources/com/google/api/codegen/py/standalone_sample.snip

```diff
@@ -74,8 +74,14 @@
 
   client = {@apiMethod.apiModuleName}.{@apiMethod.apiClassName}()
 
+  @if sample.initCode.argDefaultLines
+    @# ARGS - These lines should be commented. The top-level vars are declared by function param.
```

Comment:
They should be commented here, but used at the call site to set the actual parameters.

Reply:
The sample file will be copy-paste runnable, so it will have a main that will call into the sample function. In Goish pseudocode ("Gpc"?):

```
import (
...
)


// GetBookSample retrieves bookName
//   bookName: Name of book to retrieve
// [START sample]
func GetBookSample(bookName string) {


  // [START sample_core]

  client := ...
  request := ...
 
  // bookName = "War and Peace"   
  request.title = bookName

   response := client.GetBook(request)
   fmt.Print(response.pages) 

  // [END sample_core]
}

// [END sample]

func main() {
   myBook := flagValue("book_name", "Name of book to retrieve", "War and Peace")
   invokeExample(myBook)
}

```

The whole file is runnable. The part between the `SAMPLE` tags will be displayed in DPE docs. The part between the `SAMPLE_core` tags will be displayed in APIx. 

Given that we include the function signature (as per [existing precedent](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/language/cloud-client/v1/snippets.py)) I would leave out the line with the commented-out `bookName` sample assignment, but my understanding is that at the moment DPE wants it in there so it's super clear. 

Regardless, we may wind up tweaking the location of the tags, but this code structure reflects my understanding of the agreed solution to the requirements.

Reply:
Ah that makes sense. Is there any necessary change to this PR then? I'm not sure how to comment things out yet :/ I'm hoping to do that in a follow up if you're OK with it.

Reply:
Some languages (I assume all) have template functions to comment out lines of code, so we can use that. I'm fine if you want to defer this to the chaser PR, but add a `FIXME` line here to say that change will be coming shortly.